plugins/module_utils/openssh/certificate.py

```diff
@@ -174,6 +203,78 @@ def _time_string_to_datetime(time_string):
         return result
 
 
+class OpensshCertificateOption(object):
+    def __init__(self, option_type, name, data):
+        if not isinstance(option_type, six.string_types) or option_type not in ('critical', 'extension'):
```

Comment:
This condition is stricter than the other:
```suggestion
        if option_type not in ('critical', 'extension'):
```